apps/redi-talent-pool/src/pages/app/me/MeJobseeker.tsx

```diff
@@ -46,6 +46,12 @@ export function MeJobseeker() {
       isProfileVisibleToCompanies: !profile.isProfileVisibleToCompanies,
     })
 
+  const onJobFair2023ParticipateChange = () =>
+    mutation.mutate({
+      ...profile,
+      isJobFair2023Participant: !profile.isJobFair2023Participant,
```

Comment:
Just quickly reviewing, so didn't have time to check: don't we use (other places in the TP codebase) ways to do _partial_ updates? (kind of PUT vs. PATCH). That would be more light-weight and just cleaner.